rpcs3/Input/hid_pad_handler.cpp

```diff
@@ -56,9 +61,9 @@ struct hid_instance
 
 		hid_log.notice("Initializing HIDAPI ...");
 
-		if (hid_init() != 0)
+		if (int errorCode = hid_init(); errorCode != 0)
 		{
-			hid_log.fatal("hid_init error");
+			hid_log.fatal("hid_init error %d", errorCode);
```

Comment:
```suggestion
			hid_log.fatal("hid_init error: %s", hid_error(errorCode));
```

Reply:
hid_error accepts hid_device, not error code

Reply:
That's weird because my source code tells me to use it for hid_init.
But I can only double check tomorrow